ios/FluentUI/People Picker/ContactView.swift

```diff
@@ -199,4 +215,35 @@ open class ContactView: UIView {
 
         subtitleLabel = label
     }
+
+    private func setupPressedStateOverlay() {
+        pressedStateOverlay.clipsToBounds = true
+        pressedStateOverlay.frame = avatarView.frame
+        pressedStateOverlay.layer.cornerRadius = avatarView.frame.width / 2
+    }
+
+    private struct Constants {
+        static let labelMinimumHeight: CGFloat = 16.0
+        static let titleLabelMaximumHeight: CGFloat = 28.0
+        static let subtitleMaximumHeight: CGFloat = 24.0
+        static let spacingBetweenAvatarAndLabelContainer: CGFloat = 13.0
+        static let numberOfLinesForSingleLabel: Int = 2
+    }
+
+    open override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        super.touchesBegan(touches, with: event)
+        pressedStateOverlay.backgroundColor = Colors.Contact.pressedState
+        pressedStateOverlay.isHidden = false
+    }
+
+    open override func touchesMoved(_ touches: Set<UITouch>, with event: UIEvent?) {
+        super.touchesMoved(touches, with: event)
+        pressedStateOverlay.isHidden = true
+    }
+
+    open override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+        super.touchesEnded(touches, with: event)
```

Comment:
I don't think that the touchesEnded is called necessarily because a touchUpInside event happened.
Generally, a tap event is considered a touch up inside event meaning that the touch up event happened within the bounds of the view. Let's make sure that the tap event is only triggered when the user lifts his finger above the view (or follow whatever Outlook is doing if it differs).

Reply:
Got it. Matching Outlook requires us to use `touchUpInside`.